src/expandableCalendar/agendaList.js

```diff
@@ -115,16 +115,16 @@ class AgendaList extends Component {
     _.invoke(this.props, 'onMomentumScrollEnd', event);
   }
 
-  onHeaderLayout = ({nativeEvent}) => {
+  onHeaderLayout = ({ nativeEvent }) => {
     this.sectionHeight = nativeEvent.layout.height;
   }
 
-  renderSectionHeader = ({section: {title}}) => {
-    const today = XDate().toString(this.props.dayFormat).toUpperCase();
-    const date = XDate(title).toString(this.props.dayFormat).toUpperCase();
+  renderSectionHeader = ({ section: { title } }) => {
```

Comment:
I thought we were going to go something along the lines of this, so we could fully override the rendering of the header:
```javascript
  renderSectionHeader = ({ section: { title, renderSectionHeader } }) => {

  if(renderSectionHeader) return renderSectionHeader(title);
```
or even add a `renderSectionheader` prop and call that instead of this function.

or a combination
```
renderSectionHeader = ({ section }) => {
if(this.props.renderSectionHeader) this.props.renderSectionHeader(section);
const { title } = section;

```

Reply:
I try to avoid giving render props. If gives us a lot of customizability, but it also allows us to be non-standard with our usage of the library.

I will add it.